rmc/resources/basics.py

```diff
@@ -1,405 +1,109 @@
-"""
-Path to bucket containing resources related to building MPC (missense badness, Polyphen-2, and Constraint) score.
-"""
-
-constraint_prep = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(
-            path=f"{MODEL_PREFIX}/{version}/context_obs_exp_annot.ht"
-        )
-        for version in GNOMAD_VERSIONS
-    },
-)
-"""
-Context Table ready for RMC calculations.
-
-HT is annotated with observed and expected variant counts per base.
-"""
-
-hi_genes = f"{RESOURCE_PREFIX}/HI_genes.rCNV.txt"
-"""
-Path to haploinsufficient genes that cause severe disease.
-
-List is from Ryan Collins.
-"""
-
-## Constraint related resources
-CONSTRAINT_PREFIX = f"{RMC_PREFIX}/constraint"
-one_break = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(path=f"{CONSTRAINT_PREFIX}/{version}/one_break.ht")
-        for version in GNOMAD_VERSIONS
-    },
-)
-"""
-Table containing transcripts with at least one break.
-
-Found when searching constraint_prep HT for transcripts for a single (first) break.
-"""
-
-not_one_break = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(path=f"{CONSTRAINT_PREFIX}/{version}/not_one_break.ht")
-        for version in GNOMAD_VERSIONS
-    },
-)
-"""
-Table containing transcripts without one significant break.
-
-Transcripts in this table will be processed to check for two simultaneous breaks.
-"""
-
-not_one_break_grouped = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        CURRENT_VERSION: TableResource(
-            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/not_one_break_grouped.ht"
-        ),
-    },
-)
-"""
-Not one break Table grouped by transcript with observed missense, expected missense, and positions collected into lists.
-
-Input to searching for simultaneous breaks.
-"""
-
-multiple_breaks = VersionedTableResource(
-    default_version=CURRENT_VERSION,
-    versions={
-        version: TableResource(path=f"{CONSTRAINT_PREFIX}/{version}/multiple_breaks.ht")
-        for version in GNOMAD_VERSIONS
-    },
-)
-"""
-Table containing transcripts with multiple breaks.
-"""
+Path to bucket containing resources related to building the mutational models.
 
-simul_break_under_threshold = (
-    f"{MODEL_PREFIX}/{CURRENT_VERSION}/transcripts_under_5k.he"
-)
-"""
-SetExpression containing transcripts with fewer possible missense positions than cutoff specified in `run_simultaneous_breaks.py`.
-"""
-
-simul_break_over_threshold = f"{MODEL_PREFIX}/{CURRENT_VERSION}/transcripts_over_5k.he"
-"""
-SetExpression containing transcripts with greater than or equal to the cutoff for possible missense positions.
+Data in this bucket is used to set up regional missense constraint (RMC) calculations.
 """
 
-simul_break_temp = f"{temp_path}/simul_breaks"
+SIMUL_BREAK_TEMP = f"{TEMP_PATH}/simul_breaks"
```

Comment:
```suggestion
SIMUL_BREAK_TEMP_PATH = f"{TEMP_PATH}/simul_breaks"
```
And subsequent changes in other files, just for consistency with `TEMP_PATH`